Appsilon/shiny.fluent
# Add `manifest.json` for the showcase dashboard

### Changes
Add a `manifest.json` file generated with `rsconnect::writeManifest()` so that we can use git-backed deployment for the showcase dashboard. This way it will be updated automatically when we push to `master`.

### How to test
Check out the test deployment [here](https://connect.appsilon.com/connect/#/apps/9fa335a8-0a27-4c64-9ca0-2d76630b1237/access). After this PR is merged I'll replace the [actual deployment](https://connect.appsilon.com/fluentui/) used on our demo site with this one.
